lesson/outline/outline-template.md

```diff
@@ -1,104 +1,323 @@
+
+### Which course outcomes will be covered by this exercise?
+
+1. Learners will be able to install Jest into a workplace and also configure jest with the ‘test’ script.
+
+
+### Narrative Summary
+
+1. Introduce the project that will be tested 
+2. Outline the steps to set up the workplace and  install jest
+3. Explain file labeling and structure to best use jest 
+4. Write test script
+
 
 ### Checkpoints Summary
 
-_Check out the [content standards](http://curriculum-documentation.codecademy.com/Content-Standards/checkpoint/) for guidance on writing narratives for exercises._
+1. Initialize npm, creating a package.json file
+2. Run the “jest  myproject.test.js” command
+3. Edit package.json to include test script
+4. Run ‘npm test’ command 
 
-1. Checkpoint one
 2. Checkpoint two
 
 #### What is the purpose of these checkpoints?
 
-
+- The learner will be able to set up a basic jest testing environment
 
 ### What would you like to have in the workspace for this exercise? Share your plan below.
 
-_Check out the [content standards](http://curriculum-documentation.codecademy.com/Content-Standards/workspaces/) for guidance on writing narratives for exercises._
-
+- I.E. text editor and output terminal
 <hr>
 
-## Exercise # 2: _Insert exercise title here_
+## Exercise # 3: The `test()` function
 
 ### Which course outcomes will be covered by this exercise?
 
-1. Learners will be able to...
+1. Learners will be able to begin understanding the basic syntax involved with writing tests in jest (Ex. `test()` function).
+
 
 ### Narrative Summary
 
-_Check out the [content standards](http://curriculum-documentation.codecademy.com/Content-Standards/narrative/) for guidance on writing narratives for exercises._
+1. Introduce the test() function 
+2. Explain the three test function parameters.
+    - test(‘description of desired behavior’) 
+    - test(‘description of desired behavior. ()=>{})
+    - test(‘description of desired behavior. ()=>{}, 3000)	
+
+
+### Checkpoints Summary
+
+
+1. Create an empty test function in the myproject.test.js file using the syntax covered above. 
+2. Describe the test 
+3. Declare the function using the default timeout 
+
+
+#### What is the purpose of these checkpoints?
+
+- The learner will be able to set up a test() function 
+
+### What would you like to have in the workspace for this exercise? Share your plan below.
+
+- I.E. text editor and output terminal
+
+## Exercise # 4: The `expect()` function
+
+### Which course outcomes will be covered by this exercise?
+
+1. Learners will be able write a basic unit test in jest using the expect() function in conjunction with five of types of matchers. 
+- Five types of matchers.
+    - .toBe and .toEqual
+    - .toContain
+    - .toBeTruthy
+    - .not
+    - .toBeDefined 
+
+
+### Narrative Summary
+
+1. In this lesson, we will cover the necessary basic syntax for writing our first complete unit test inside of our previously constructed test() function.
```

Comment:
```suggestion
1. In this exercise, we will cover the necessary basic syntax for writing our first complete unit test inside of our previously constructed test() function.
```

Reply:
Make sure that you also define what a `matcher` is since this is a new vocabulary term